app/aragonRedux/aragonRedux.js

```diff
@@ -1,7 +1,10 @@
 import _ from 'lodash'
 
-export const differenceWithEvents = (initialEventState, finalEventState) => {
-  return _.differenceWith(finalEventState, initialEventState, (a, b) => {
+export const filterNewEvents = (initialEventState, finalEventState) => {
```

Comment:
what is this for?

Reply:
aragon's event observable subscribes sometimes give the same event more than once, so this checks the tx hash to filter out any events that our reducer's have already seen

Reply:
lol